scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,157 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+    group_role TEXT;
+BEGIN
+    group_role := cartodb._CDB_Group_GroupRole(group_name);
+    PERFORM cartodb._CDB_Group_CreateGroup_API(current_database(), group_name, group_role);
```

Comment:
Uhm, it's needed because it's the creation of the group server-side, this is what keeps both sides on sync :_)

It's a creation (a post to a rest endpoint), so I think this naming more accurate.

@javisantana I think you were just joking about the naming, isn't it? :_D PG notifications occur after the transaction, so it doesn't fit our needs.

I just realised it should happen role after creation, moving...